File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/adma/ADM107ULayDownListInfo.java
package nta.med.data.model.ihis.adma;

public class ADM107ULayDownListInfo {
	private String userId;       
	private String sysId;        
	private String trId;         
	private Double trSeq;        
	private String pgmId;        
	private String upprMenu;     
	private String pgmNm;
	private String pgmTp;
	private String useYn;
	public ADM107ULayDownListInfo(String userId, String sysId, String trId,
			Double trSeq, String pgmId, String upprMenu, String pgmNm,
			String pgmTp, String useYn) {
		super();
		this.userId = userId;
		this.sysId = sysId;
		this.trId = trId;
		this.trSeq = trSeq;
		this.pgmId = pgmId;
		this.upprMenu = upprMenu;
		this.pgmNm = pgmNm;
		this.pgmTp = pgmTp;
		this.useYn = useYn;
	}
	public String getUserId() {
		return userId;
	}
	public void setUserId(String userId) {
		this.userId = userId;
	}
	public String getSysId() {
		return sysId;
	}
	public void setSysId(String sysId) {
		this.sysId = sysId;
	}
	public String getTrId() {
		return trId;
	}
	public void setTrId(String trId) {
		this.trId = trId;
	}
	public Double getTrSeq() {
		return trSeq;
	}
	public void setTrSeq(Double trSeq) {
		this.trSeq = trSeq;
	}
	public String getPgmId() {
		return pgmId;
	}
	public void setPgmId(String pgmId) {
		this.pgmId = pgmId;
	}
	public String getUpprMenu() {
		return upprMenu;
	}
	public void setUpprMenu(String upprMenu) {
		this.upprMenu = upprMenu;
	}
	public String getPgmNm() {
		return pgmNm;
	}
	public void setPgmNm(String pgmNm) {
		this.pgmNm = pgmNm;
	}
	public String getPgmTp() {
		return pgmTp;
	}
	public void setPgmTp(String pgmTp) {
		this.pgmTp = pgmTp;
	}
	public String getUseYn() {
		return useYn;
	}
	public void setUseYn(String useYn) {
		this.useYn = useYn;
	}
	
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Adma/Adm107ULayDownListInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Adma
{
    [Serializable]
    public class Adm107ULayDownListInfo
    {
        private String _userId;
        private String _sysId;
        private String _trId;
        private String _trSeq;
        private String _pgmId;
        private String _upprMenu;
        private String _pgmNm;
        private String _pgmTp;
        private String _useYn;

        public String UserId
        {
            get { return this._userId; }
            set { this._userId = value; }
        }

        public String SysId
        {
            get { return this._sysId; }
            set { this._sysId = value; }
        }

        public String TrId
        {
            get { return this._trId; }
            set { this._trId = value; }
        }

        public String TrSeq
        {
            get { return this._trSeq; }
            set { this._trSeq = value; }
        }

        public String PgmId
        {
            get { return this._pgmId; }
            set { this._pgmId = value; }
        }

        public String UpprMenu
        {
            get { return this._upprMenu; }
            set { this._upprMenu = value; }
        }

        public String PgmNm
        {
            get { return this._pgmNm; }
            set { this._pgmNm = value; }
        }

        public String PgmTp
        {
            get { return this._pgmTp; }
            set { this._pgmTp = value; }
        }

        public String UseYn
        {
            get { return this._useYn; }
            set { this._useYn = value; }
        }

        public Adm107ULayDownListInfo() { }

        public Adm107ULayDownListInfo(String userId, String sysId, String trId, String trSeq, String pgmId, String upprMenu, String pgmNm, String pgmTp, String useYn)
        {
            this._userId = userId;
            this._sysId = sysId;
            this._trId = trId;
            this._trSeq = trSeq;
            this._pgmId = pgmId;
            this._upprMenu = upprMenu;
            this._pgmNm = pgmNm;
            this._pgmTp = pgmTp;
            this._useYn = useYn;
        }

    }
}